### PR TITLE
feat: frontend fixes

### PR DESCRIPTION
**Pull request by kkpeckk**

### CODE REVIEW

File: src/components/Inflenser/AdvertisersStats.js [hunk 1]
```diff
@@ -29,25 +29,25 @@ export default function AdvertisersStats() {
             <Table size="sm" variant="simple">
               <Thead>
                 <Tr borderColor="gray.500">
-                  <Th fontSize={16} color="#1A4587" fontFamily="'Prompt', sans-serif">
+                  <Th fontSize={16} color="#1A4587" fontFamily="'Prompt', sans-serif" textTransform="none">
```
Comment (luduvigo) [inline src/components/Inflenser/AdvertisersStats.js]: Can we reorganise this part and avoid setting the style for every `Th` tag?

File: src/components/Inflenser/AdvertisersStats.js [hunk 2]
```diff
@@ -80,11 +80,11 @@ const CampaignsStatsTab = ({ ...props }) => {
         <Td fontSize={15} color="#00203F" fontWeight={600}>
           Fan Token summer special promotion
         </Td>
-        <Td fontWeight={300}>$ {campaign.earned / 1e6} </Td>
-        <Td fontWeight={300}>{campaign.mirrors}</Td>
-        <Td fontWeight={300}>{campaign.clicks.toNumber()}</Td>
-        <Td fontWeight={300}>{campaign.actions.toNumber()}</Td>
-        <Td fontWeight={300}>$ {campaign.earned / 1e6}</Td>
+        <Td fontWeight={400}>$ {campaign.earned / 1e6} </Td>
```
Comment (luduvigo) [inline src/components/Inflenser/AdvertisersStats.js]: Same here, not the best practice to set fontWeight to 400 for every tag

File: src/components/Inflenser/PostCard.js [hunk 3]
```diff
@@ -454,6 +464,16 @@ export default function PostCard({ publicationId }) {
                     w={{ base: '100%', md: '38%' }}
                     h="auto"
                     onClick={() => handleCreatePost()}
+                    fontFamily="'Prompt', sans-serif"
```
Comment (luduvigo) [inline src/components/Inflenser/PostCard.js]: Seen this style in several places, can we set it up in one place?

File: src/components/layout/HowItWorks.js [hunk 4]
```diff
@@ -28,7 +28,7 @@ export default function HowItWorks() {
           Lens Booster is for users of Lens protocol and Lens applications such as Lenster.xyz
         </Text>
       </Box>
-      <Box bg="#ffffff" borderRadius="20px" p={6} textAlign="left" mt={6} boxShadow="lg" fontWeight="300">
+      <Box bg="#ffffff" borderRadius="20px" p={6} textAlign="left" mt={6} boxShadow="lg">
```
Comment (luduvigo) [inline src/components/layout/HowItWorks.js:31]: This one is duplicated as well

File: src/components/navbar/Navbar.js [hunk 5]
```diff
@@ -23,12 +23,15 @@ import {
...
 import { Link as LinkRouter } from 'react-router-dom';
 
+import LogoLight from '../icons/LogoLight';
 import LensBoosterIcon from '../icons/LensBoosterIcon';
 import Discord from '../icons/Discord';
 import Twitter from '../icons/Twitter';
 import Lens from '../icons/Lens';
 import Connect from './Connect.js';
 import SelectProfile from './SelectProfile.js';
+import { useEffect, useState } from 'react';
```
Comment (luduvigo) [inline src/components/navbar/Navbar.js]: Let's please put external imports (imports from other projects) on top and internal imports (imports from our project) below.

File: src/components/navbar/Navbar.js [hunk 6]
```diff
@@ -82,16 +78,16 @@ export default function Navbar() {
             {isLargerThan640 && (
               <>
                 <Box mr={3}>
-                  <Link href="https://twitter.com/LensBooster">
+                  <Link href="https://twitter.com/LensBooster" fill="#5C6F81" _hover={{ fill: '#FF6827' }}>
```
Comment (luduvigo) [inline src/components/navbar/Navbar.js]: Can we set it up in one place as well?

File: src/hooks/usePublication.js [hunk 7]
```diff
@@ -39,6 +39,8 @@ export const fetchPublication = async (publicationId) => {
         );
         arrayJsxPost.push(
           <Link
+            fontWeight={600}
```
Comment (luduvigo) [inline src/hooks/usePublication.js:42]: Same as previous comments

**Review by luduvigo — CHANGES_REQUESTED** [review]

File: src/components/Inflenser/PostCard.js [hunk 8]
```diff
@@ -155,9 +155,7 @@ export default function PostCard({ publicationId }) {
                   }
                 />
                 <Box marginTop="auto" marginBottom="auto" marginLeft="1rem" fontSize="16px">
```
Comment (luduvigo) [inline src/components/Inflenser/PostCard.js:157]: Can you check why Github is not getting JS here?

File: src/components/Inflenser/PostCard.js [hunk 9]
```diff
@@ -155,9 +155,7 @@ export default function PostCard({ publicationId }) {
...
-                  <Text fontWeight={500} fontFamily="'Prompt', sans-serif">
-                    {publication.profile.name}
-                  </Text>
+                  <Text variant="inflenserPageTitles">{publication.profile.name}</Text>
```
Comment (luduvigo) [inline src/components/Inflenser/PostCard.js:158]: Can we improve the naming of `arrayJsxPost2` ?

File: src/components/footer/Footer.js [hunk 10]
```diff
@@ -1,95 +1,209 @@
...
+                <Box w="fit-content" my="auto" mr={2}>
+                  <FiveElementsFooter />
+                </Box>{' '}
+                <Text color="#5C6F81" width="fit-content" display="inline" fontSize={15}>
+                  By Five Elements Labs
+                </Text>
+              </Flex>
+            </Link>
+          </GridItem>
+          <GridItem colSpan={{ base: '6', md: '2' }}>
```
Comment (luduvigo) [inline src/components/footer/Footer.js:45]: Can't we add this `GridItem` to the Theme?

File: src/components/navbar/Navbar.js [hunk 11]
```diff
@@ -71,38 +86,209 @@ export default function Navbar() {
...
-                  </Box>
-                </Link>
-              </>
-            )}
             <Spacer display={{ base: 'initial', md: 'none' }} />
             <Flex alignItems="center">
               {' '}
               <Connect variant="solid">Connect Wallet</Connect>
+              {isLargerThan640 && (
+                <Box>
```
Comment (luduvigo) [inline src/components/navbar/Navbar.js:98]: Looks like a lot of repeated code
Can't we put part of it in the theme?

**Review by luduvigo — CHANGES_REQUESTED** [review]

File: src/components/footer/Footer.js (hunk 10 above)
Comment (AlbyGiaco) [reply src/components/footer/Footer.js:45]: chakra can't do that

**Review by luduvigo — APPROVED** [review]

**Pull request merged**